src/codecs/html/README.md

```diff
@@ -38,3 +38,126 @@ Some of the main external dependencies:
 - `js-beautify` for pretty generated HTML.
 
 - `json5` for more human readable representation of arbitrary values.
+
+---
+
+## Microdata and semantic tag usage (WIP)
+
+### Schema nodes to add microdata to
+* Article
+    * `div` `itemtype='schema:Article'`
+    * itemprops:
+        * authors* (author?): `ol`
+        * title*: `h1`? (+ `title`)
+        * environment: `data`?
+* Audio
+    * `audio`
+    * itemprops:
+        * caption*: `figcaption`
+        * transcript
+* Brand
+    * `picture`/`img` `itemtype='schema:Brand'`
+    * itemprops:
+        * name: maybe unnecessary, just use in alt tag?
+        * logo*: `source`/`image`
+        * reviews
+* Code
+    * `div` `itemtype='stencila:Code'`
```

Comment:
I'd propose using the [`<code> tag`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code)

Reply:
Oh! I think this was the redundant example I was trying to explain yesterday. It slipped my mind that we could set props to the encoded tag's contents instead of storing it as another tag with itemprop. 🤦‍♀ I'll go ahead and update this one.